webapp/src/main/java/com/box/l10n/mojito/rest/asset/AssetWS.java

```diff
@@ -95,7 +95,7 @@ public class AssetWS {
   @RequestMapping(value = "/api/assets", method = RequestMethod.GET)
   public List<Asset> getAssets(
       @RequestParam(value = "repositoryId") Long repositoryId,
-      @RequestParam(value = "path", required = false) String path,
+      @RequestParam(value = "assetPath", required = false) String path,
```

Comment:
This param value causes an error when generating a client using RestTemplate library